### ISSUE TITLE
Bug: Stringtie - merge  (v0.1.0)

### ISSUE THREAD

**Issue by MoHeydarian** [galaxyproject/tools-iuc#1225]
This wrapper allows the user to select multiple single input_gtf files (generated by Stringtie or Cufflinks), though when the input_gtf files are generated using collections, the user can only select a single collection to "merge". 

This can be fixed by adding the "Additional GTF inputs (Lists)" feature that Cuffmerge uses, which allows selecting multiple collections of GTF files as input to "merge".

**Comment by nekrut** [comment]
is this when you want to marge a collection and several individual GFFs?

**Comment by MoHeydarian** [comment]
Yes, that would be one use case, where you have Stringtie reconstructions in collections and as individual datasets. This currently won't work with the way the input selection is handled in the wrapper. 

In a cleaner example, like your recent [RNAseq tutorial](https://galaxyproject.org/tutorials/nt_rnaseq/) where you have two collections, each representing an experimental condition (G1E and mega) that you would like to merge with a reference, you would only be able to input one or the other the way the input selection is set up. I see in your tutorial you combined the collections into one so you could use Stringtie-merge. If this wrapper used the "Additional GTF inputs (Lists)" feature that Cuffmerge uses, you could eliminate the additional step to combine the reconstructions upstream of Stringtie merge.

**Comment by nekrut** [comment]
I see

**Comment by bgruening** [comment]
@MoHeydarian do you have more information about this "Additional GTF inputs (Lists)" option? I can put this into my recent stringtie work if you think this is ok.

**Comment by MoHeydarian** [comment]
@bgruening I can point you to the Cuffmerge wrapper (it seems like a complicated one to me :( ), I'm not sure where in there the option to add more GTF inputs is.  Yes, I think this would be a great addition to your Stringtie update!

**Comment by bgruening** [comment]
@MoHeydarian I'm confused, stringtie --merge is it's own program. How does it belong to cuffmerge.

**Comment by MoHeydarian** [comment]
Oh sorry for not being clear, I was suggesting taking the feature of the Cuffmerge wrapper that allows to add "Additional GTF inputs" and putting it in the Stringtie-merge wrapper.

**Comment by jmchilton** [comment]
cuffmerge does have that hack and I wrote it because I felt this was needed at the time. But since then I had believed I added the ability to select multiple collections when doing a reduction in Galaxy with https://github.com/galaxyproject/galaxy/pull/805. However I just retested this and it doesn't seem to work anymore - I don't seem to be able to select multiple collections - I guess this was broken at some point and noone noticed.

So we can hack up stringtie merge like cuffmerge or we can fix the Galaxy tool form.
